### PR TITLE
Fix missing trailing whitespace on intl. locales

### PR DESCRIPTION
**Pull request by guixxx**
If you're not using an en-US locale, pacaur doesn't add a trailing
whitespace at the end of messages that require user input. It's a
silly thing, but it looks ugly and I just decided to fix it.

### CODE REVIEW

**Comment by rmarquis** [comment]
Thanks. Do you have any idea why this specific issue arises? I've tried to look for relevant bug reports/similar issues, but haven't found anything.

**Comment by guixxx** [comment]
Upon further investigation, I've realized that this behavior has nothing to do with `pacaur`.

Before I sent this PR, I was trying to fix this issue by only editing the `.po` file for my language (`pt.po`), I went ahead and added the [missing trailing whitespace](https://github.com/rmarquis/pacaur/blob/4.7.6/po/pt.po#L363), and that's where I messed up, because whenever I ran this: 

```
$ LC_ALL=pt_PT.UTF-8 ./pacaur -S pacaur
:: Package pacaur not found in repositories, trying AUR...
:: resolvendo dependências...
:: procurando por conflitos internos...
:: pacaur-4.7.6-1 está atualizado -- reinstalando

AUR Packages  (1) pacaur-4.7.6-1  

:: Continuar instalação? [S/n]n
$ _
```

I wasn't actually getting an updated message, because I forgot to edit the *uppercase* string at [line 359](https://github.com/rmarquis/pacaur/blob/4.7.6/po/pt.po#L359). 🙄

**TL;DR:** My _honest_ mistake. The real issue is that some translators forget to add a whitespace at the end of the `msgid` string. Nevertheless, I do think this PR is still valid, because now the trailing whitespace is separated from the `msgid` string, so translators won't have to append a whitespace anymore. It's totally up to you if you don't want to change it, of course.

Thank you and I'm sorry for the confusion!

**Comment by rmarquis** [comment]
I see. I wouldn't be against fixing that space issue, but there are quite a few other locations that have space (trailing or leading). I really don't think investing time to fix this everywhere is worth it, nor do I think the empty double quotation marks is cleaner. As such, I'll simply fix the PT translation in a quick fix.

**Comment by rmarquis** [comment]
Done in 598ef0cb4545ff181e382e7eeb9b49d79d51b250.
